packages/core/package.json

```diff
@@ -136,6 +136,7 @@
     "json-stringify-safe": "^5.0.1",
     "langchain": "0.3.6",
     "lodash": "^4.17.21",
+    "node-gzip": "^1.1.2",
```

Comment:
Required for decompressing the verification response and eventual flat file response from Surescripts.